src/containers/AccountConnection.jsx

```diff
@@ -50,12 +50,12 @@ class AccountConnection extends Component {
     const folderField = connector && connector.fields && connector.fields.folderPath
     if (!folderField) return null
     const auth = account && account.auth
-    if (!folderField.default && auth && !auth.folderPath) {
-      return t('account.config.default_folder', connector)
-    }
     if (folderField.default) {
       return folderField.default
     }
+    if (!(auth && auth.folderPath)) {
+      return t('account.config.default_folder', connector)
+    }
     if (auth && auth.folderPath) {
       return account.folderPath
```

Comment:
@CPatchane I am asking myself if this line should not be `return auth.folderPath` instead ?